src/systems/a320_systems_wasm/src/nose_wheel_steering.rs

```diff
@@ -42,7 +42,7 @@ pub(super) fn nose_wheel_steering(builder: &mut MsfsAspectBuilder) -> Result<(),
     // Lacking a better event to bind to, we've picked a mixture axis for setting the
     // tiller handle position.
```

Comment:
Should these comments be removed?

Reply:
It seems that they just added a new and distinct axis, but the remaining behavior seems to be the same. Therefore we still need it.

Reply:
I agree with @pepperoni505. I think the comment should be removed, as we're no longer using the "mixture axis" when this change is incorporated.

```suggestion
```

Reply:
Agree...done